lsp/nls/src/requests/completion.rs

```diff
@@ -729,11 +731,27 @@ pub fn handle_completion(
         index: (cursor.index.0.saturating_sub(1)).into(),
         ..cursor
     };
+    let trigger = params
+        .context
+        .as_ref()
+        .and_then(|context| context.trigger_character.as_deref());
 
     let term = server.lookup_term_by_position(pos)?.cloned();
+
+    if let Some(Term::Import(import)) = term.as_ref().map(|t| t.term.as_ref()) {
+        // Don't respond with anything if trigger is a `.`, as that may be the
+        // start of a relative file path `./`, or the start of a file extension
```

Comment:
What's wrong with completing the file extension?

Reply:
If we send a new completion while trying to complete the file extension (something like: `thing.<cursor>`, then if the user tab-completes the new completion suggestion it will end up as `thing.thing.ncl`

There are maybe some ways around this, like using `EditText` instead of insert or only giving the file extensions as completion, but by just ignoring the `.` it will continue as normal and `thing.` gets tab completed to `thing.ncl`